build/common.mk

```diff
@@ -36,11 +36,9 @@
 #
 #  - only_build: this target runs build and install targets
 #
-#  - only_test: this target runs the lint and test_all targets
+#  - only_test: this target runs the list and test_all targets
```

Comment:
is that correct?

Reply:
I'm not sure how this happened, I copied all of this from other repos...